SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/controller/SubscriptionController.java

```diff
@@ -103,7 +103,7 @@ public Uni<RestResponse<Object>> getSubscriptionById(HttpServerRequest request,
 			return subService.getSubscription(HttpUtils.getTenant(request), subscriptionId).onItem()
 					.transformToUni(subscription -> {
 						return HttpUtils.generateEntityResult(contextHeader, context, acceptHeader, subscription, null,
-								options, null, ldService);
+								options, null, ldService,null,null);
```

Comment:
Add postman test cases for Pick and omit functionality.

Reply:
done